runtime/libia2/include/ia2.h

```diff
@@ -161,6 +161,8 @@
 #define IA2_CAST(func, ty) (ty) { (void *)IA2_FN_ADDR(func) }
 #endif // !IA2_ENABLE
 
+#define MAX_PKEYS 16
```

Comment:
We should namespace things in this header (i.e. IA2_MAX_PKEYS). Also that's x86-specific so maybe `IA2_MAX_COMPARTMENTS` is better.